packages/core/bootstrap/src/lib/middleware/burst-limit/index.ts

```diff
@@ -69,6 +69,9 @@ export const withBurstLimit =
         observedRequestsInMinute > config.burstCapacity1m * MINUTE_LIMIT_WARMER_BUFFER
         // TODO: determine BATCH_REQUEST_BUFFER dynamically based on (number of batch warmers * 3)
       ) {
+        if (config.enabled) {
+          logger.warn(`External Adapter custom rate limit capacity is being used`)
+        }
         logger.warn(
           `External Adapter backing off. Provider's limit of ${
```

Comment:
Can we combine these in one log line?

```suggestion
        logger.warn(
          `${ customCapacitySet ? 'CUSTOM RATE LIMIT CAPACITY CONFIGURED: ' : '' }External Adapter backing off. Provider's limit of ${
```